src/components/Tweet.js

```diff
@@ -1,34 +1,25 @@
-function Tweet() {
+import ProfileImage from "./ProfileImage";
+import User from "./User";
+import Timestamp from "./Timestamp";
+import Message from "./Message";
+import Actions from "./Actions";
+
+function Tweet({tweet}) {
+  const { user, timestamp, message } = tweet;
```

Comment:
Here, you could've passed the tweet as props, like:

`function Tweet(props) {
  const {tweet} = props;`